lib/Genesis/CI/Legacy.pm

```diff
@@ -610,7 +610,9 @@ pipeline:
     repo:        (( param "Please specify the name of the Git repository" ))
     branch:      master
     private_key: (( param "Please generate an SSH Deployment Key and install it into Github (with write privileges)" ))
-
+  vault:
+    role:   (( vault "secret/exodus/ci/genesis_pipelines:approle_id" ))
+    secret: (( vault "secret/exodus/ci/genesis_pipelines:approle_secret" ))
```

Comment:
The outstanding item on the other related PR is to rename `genesis_pipelines` to `genesis-pipelines`; we'll need to remember to propagate the update here as well.